src/site/site.xml

```diff
@@ -77,7 +77,7 @@ under the License.
       <item name="Parent POMs"       href="https://maven.apache.org/pom/index.html" />
       <item name="Plugins"           href="https://maven.apache.org/plugins/index.html" />
       <item name="Skins"             href="https://maven.apache.org/skins/index.html" />
-      <item name="">
```

Comment:
I don't think "Components" is the _best_ term because this section also contains "Shared Components", but I couldn't think of a better name.

If anyone has ideas please suggest them!